lib/tests/streamlit/runtime/caching/hashing_test.py

```diff
@@ -291,6 +291,23 @@ class EnumClass(Enum):
 
         assert get_hash(enum_1)
```

Comment:
I know this is from a previous PR, but this test is lacking as well. See my comment about `assert get_hash()` being strange -- but also, one of the big things we care about testing with hashing is that _the hashing function returns the same result when called with the same result_. So I think this should be modified to something like:

```python
# Hash values should be stable
self.assertEqual(get_hash(EnumClass.ENUM1), get_hash(EnumClass.ENUM1))

# Different enum values should produce different hashes
self.assertNotEqual(get_hash(EnumClass.ENUM1), get_hash(EnumClass.ENUM2))
```